src/infra/marketplace/marketplaceCmdRepo.go

```diff
@@ -437,6 +444,43 @@ func (repo *MarketplaceCmdRepo) InstallItem(
 	)
 }
 
+func (repo *MarketplaceCmdRepo) parseUninstallCmdSteps(
+	filesToDelete []valueObject.UnixFileName,
+	installedSlug valueObject.MarketplaceItemSlug,
+) ([]valueObject.MarketplaceItemCmdStep, error) {
+	uninstallCmdSteps := []valueObject.MarketplaceItemCmdStep{}
+
+	trashDirName := installedSlug.String() + "-%installHostname%-%installUuid%"
+
+	createTrashDirToMoveFilesStep, err := valueObject.NewMarketplaceItemCmdStep(
```

Comment:
Why do you create such verbose variables man? We get that you'll move the files to that directory later. You don't need to explain every step of the way. Some of your variables are completely generic and some are overly verbose.  You gotta find the middle ground there.

Anyway, I wouldn't add these steps inside the parseUninstallCmdSteps, clear violation of the SRP. So this has to be refactored.

It's best if you separate these concerns. Have a method called uninstallFilesRemoval and pass only the installation id and let it do all the work. If the filesToRemove is empty, it will return nil. Simple as that.

Reply:
Adjusted!